### DISCUSSION TITLE
Can anyone tell me does this wireless carplay dongle have an Apple MFi certified chip in it?

### DISCUSSION THREAD

**Discussion by crazyyi** [ludwig-v/wireless-carplay-dongle-reverse-engineering#57] [Q&A]
Since many people are talking about porting it to raspberry Pi, wouldn't it be very hard to do so if a MFi certified chip is required?

**Comment by CiNcH83** [comment]
Discussed [here](https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/discussions/55#discussioncomment-643982)... just for reference...
